### PR TITLE
[16.0][MIG] calendar_event_link_base

### PR DESCRIPTION
**Pull request by cvinh**
Migration to 16.0

### CODE REVIEW

**Comment by cvinh** [comment]
@OCA/web-maintainers can you please help for review ?

File: calendar_event_link_base/models/calendar_event_link_mixin.py [hunk 1]
```diff
@@ -11,16 +11,18 @@ class CalendarEventLinkMixin(models.AbstractModel):
...
-    @api.multi
     def _compute_event_count(self):
-        for x in self.env["calendar.event"].read_group(
+        mapped_data = self.env["calendar.event"].read_group(
             [("res_model", "=", self._name), ("res_id", "in", self.ids)],
             ["id"],
             ["res_id"],
-        ):
-            self.browse(x["res_id"]).update({"event_count": x["res_id_count"]})
+        )
```
Comment (etobella) [inline calendar_event_link_base/models/calendar_event_link_mixin.py:19]: I would opt for something like:

```python
data = {x["res_id"]: x for x in mapped_data}
for record in self:
    record.event_count = data.get(record.id, {}).get("res_id_count", 0)
```

**Review by yvaucher — APPROVED** [review]
LGTM

Code review only

**Review by etobella — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**